domain/src/main/java/camp/nextstep/edu/calculator/domain/Expression.kt

```diff
@@ -18,25 +18,25 @@ class Expression(formulaString: String? = "") {
                 }
             return _formulas
         }
-    private var _formulas: List<String>
+    private var _formulas: MutableList<String>
```

Comment:
```suggestion
    private val _formulas: MutableList<String>
```

리스트를 초기화 한 뒤에 새로 값을 채워넣는 것 보다 객체를 새로 생성하는 비용이 더 큽니다..!